src/js/Root.jsx

```diff
@@ -23,6 +23,7 @@ import FAQ from "./routes/More/FAQ";
 import Friends from "./routes/Friends";
 import GetStarted from "./routes/Intro/GetStarted";
 import GuidePositionList from "./routes/Guide/GuidePositionList";
+import HowToUse from "./routes/More/howtouse";
 import Intro from "./routes/Intro/Intro";
```

Comment:
Hi @Anoninnyc, the Component file name should be "HowToUse.jsx", but the route that appears in the browser (elsewhere in the code) should be all lower case.